exchanges/bybit/bybit_wrapper.go

```diff
@@ -1798,12 +1811,8 @@ func (by *Bybit) ExtractCurrencyPair(symbol string, assetType asset.Item, reques
 	}
 	var pair currency.Pair
 	switch assetType {
-	case asset.Options, asset.USDCMarginedFutures:
-		pair, err = currency.NewPairFromString(symbol)
-		if err != nil {
-			return currency.EMPTYPAIR, err
-		}
-	case asset.Spot, asset.USDTMarginedFutures, asset.CoinMarginedFutures:
+	case asset.Options, asset.USDCMarginedFutures, asset.Spot,
+		asset.USDTMarginedFutures, asset.CoinMarginedFutures:
```

Comment:
You can probably remove this function completely now and call `MatchSymbolWithAvailablePairs` I don't think the format is necessary. It might be?

Reply:
I think I should keep this `ExtractCurrencyPair` function for certain places and use `MatchSymbolWithAvailablePairs` for sections where I can separate the format and the `MatchSymbolWithAvailablePairs` method call.